client/src/containers/projects/form/number-form-item.tsx

```diff
@@ -33,6 +34,7 @@ export default function NumberFormItem({
   formControlClassName,
   onValueChange,
   placeholder,
+  "data-testid": dataTestId,
```

Comment:
hm, I'm gonna need to understand why are we in need to introduce test ids. I mean, I get the purpose of this, but I'd like to better understand where/why the need is

Reply:
It's a common convention to use data-testid for testing purposes, as testing libraries provide specific methods like getByTestId. Also, it makes it easier to select certain elements and creates a clear separation of concerns between testing context and application code.